src/app.ts

```diff
@@ -116,15 +115,21 @@ async function app() {
 
   logger.info('Converting FSH to FHIR resources...');
   const outPackage = exportFHIR(tank, defs);
-  writeFHIRResources(outDir, outPackage, program.snapshot);
+  const useGeneratedFolder = !isLegacyIgPubContext;
+  writeFHIRResources(outDir, outPackage, program.snapshot, useGeneratedFolder);
 
   // If FSHOnly is true in the config, do not generate IG content, otherwise, generate IG content
   if (config.FSHOnly) {
     logger.info('Exporting FSH definitions only. No IG related content will be exported.');
   } else {
     const igDataPath = path.resolve(input, 'ig-data');
     logger.info('Assembling Implementation Guide sources...');
-    const igExporter = new IGExporter(outPackage, defs, igDataPath, isIgPubContext);
+    const igExporter = new IGExporter(
+      outPackage,
+      defs,
+      igDataPath,
+      isIgPubContext || isLegacyIgPubContext
```

Comment:
Is the `||` here necessary? As far as I can tell, `isLegacyIgPubContext` is only ever true when `isIgPubContext` is already true. What do you think?

Reply:
I think it is necessary.  My reading of it is that if `isIgPubContext` and `isLegacyIgPubContext` are mutually exclusive.  While they both can be `false` at the same time, they can never be `true` at the same time.

Line 73:
```
const isIgPubContext = inputFshFolder;
```

Line 75:
```
const isLegacyIgPubContext = fshFolder && !inputFshFolder;
```

One requires `inputFshFolder` and the other requires `!inputFshFolder`.